test/test_utils/sagemaker.py

```diff
@@ -33,10 +33,12 @@ class DLCSageMakerRemoteTestFailure(Exception):
 
 
 def assign_sagemaker_remote_job_instance_type(image):
-    if "tensorflow" in image:
-        return "ml.p3.8xlarge" if "gpu" in image else "ml.c4.4xlarge"
+    if "gpu" in image:
+        return "ml.p3.8xlarge"
+    elif "tensorflow" in image:
+        return "ml.c4.4xlarge"
     else:
-        return "ml.p2.8xlarge" if "gpu" in image else "ml.c4.8xlarge"
+        return "ml.c4.8xlarge"
```

Comment:
nit: Why turn this into a regular if-else block instead of the keeping it as the ternary-if operator like before?

Reply:
more clear this way in my opinion

Reply:
Sure.